src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22528.cs

```diff
@@ -0,0 +1,27 @@
+﻿# if IOS
```

Comment:
Does it make sense to run this test on Android and WinUI as well? 

We typically let these tests run on all platforms even if the original issue is only happening on iOS. This also lets us make sure that we have consistent behavior between all platforms.

If it's broken on WinUI and/or Android then log a bug and make a note of the bug inside this test.

Reply:
Okay, I have modified the script to run the test on both Android and WinUI. Could you please take a look and let me know?